examples/openvino/test_ov_examples.py

```diff
@@ -0,0 +1,147 @@
+#  Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import os
+import sys
+import tempfile
+import unittest
+from unittest.mock import patch
+
+#os.environ["CUDA_VISIBLE_DEVICES"] = ""
+
+SRC_DIRS = [
+    os.path.join(os.path.dirname(__file__), dirname)
+    for dirname in [
+        "text-classification",
+        "question-answering",
+        "audio-classification",
+        "image-classification",
+    ]
+]
+sys.path.extend(SRC_DIRS)
+
+if SRC_DIRS is not None:
+    import run_image_classification
+    import run_audio_classification
+    import run_glue
+    import run_qa
+
+
+class TestExamples(unittest.TestCase):
+    def test_audio_classification(self):
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            test_args = f"""
+                run_audio_classification.py  
+                --model_name_or_path facebook/wav2vec2-base  
```

Comment:
I would prefer we replace it with a tiny model :
```suggestion
                --model_name_or_path hf-internal-testing/tiny-random-Wav2Vec2Model
```